src/components/CheckboxInput/CheckboxInput.tsx

```diff
@@ -21,7 +21,7 @@ interface Props {
   /**
    * Custom content to be displayed to right of checkbox.
    */
-  label: string;
+  label: React.ReactNode;
```

Comment:
This doesn't cause an issue when a non-string is used for aria-label?

Reply:
@RyanMacBern I think you're correct here, although we have provided the ability for the team to pass nodes so that they may add things like anchor tags (which are very typical in something like a terms checkbox). Perhaps the answer here is to allow passing of these attributes separately?